code/__DEFINES/objects.dm

```diff
@@ -157,4 +157,21 @@ var/list/RESTRICTED_CAMERA_NETWORKS = list( //Those networks can only be accesse
 #define SENTRY_ALERT_DAMAGE				4
 #define SENTRY_ALERT_BATTERY			5
 #define SENTRY_ALERT_DELAY				200 //20 seconds
-#define SENTRY_DAMAGE_ALERT_DELAY		50 //5 seconds
\ No newline at end of file
+#define SENTRY_DAMAGE_ALERT_DELAY		50 //5 seconds
+
+
+//Scout cloak defines
+#define SCOUT_CLOAK_ENERGY	100
+#define SCOUT_CLOAK_STEALTH_DELAY 30
+#define SCOUT_CLOAK_RUN_DRAIN	5
+#define SCOUT_CLOAK_WALK_DRAIN	1
+#define SCOUT_CLOAK_ACTIVE_RECOVERY -5 //You only get this once every obj tick, so it'll be comparable to the inactive value
```

Comment:
I'd flip this to use positive values imo.

Reply:
Planning on doing this?

Reply:
This is a style thing; I think it's fine.